pkg/server/server.go

```diff
@@ -1319,6 +1307,27 @@ func NewServer(cfg Config, stopper *stop.Stopper) (serverctl.ServerStartupInterf
 	return lateBoundServer, err
 }
 
+// newClockFromConfig creates a HLC clock from the server configuration.
```

Comment:
Since perhaps you found find it amusing: "a" vs "an" is based on pronunciation rather than spelling, so which to use here is a little up for debate.